docs/src/components/DocEditLink.vue

```diff
@@ -0,0 +1,27 @@
+<template>
+	<div>
+		<a href="#" class="float doc-link" :href="url" target="_blank" v-if="link">
+        	Edit this Doc
+    </a>
+	</div>
+</template>
+<script>
+import docConfig from '../config/docConfig';
+	export default{
+		props: {
```

Comment:
I think we either should use 2 spaces as indent or 4 spaces not both :), it changes from file to file, it should be unified :)

Reply:
Actually dont, this have to be done in a new pull request with the goal of doing unified indention in all files :)

Either you or me can do that PR, I dont mind doing it, if you won't do it :)

Reply:
I was supposed to do it earlier with `eslint` but sticked with as it is, thought we'll do it before launch. Because everyone is working on their forked/local branches. What do you think @AlexanderYW when should we do it? I have seen many projects using `eslint`.

Reply:
Lets do it as soon this is merged, and make people merge the new version into their branch before accepting their solutions

You know there is no need to make it worse that it already is :p

Reply:
👍

Reply:
If you can do it the way you think(I'm not that familiar), I'll suggest you review this PR(I'll merge now) and look into code styling. Let me know!

Reply:
Its pretty simple, we just make a new PR after this with the purpose of correcting indentions :)